day35/Programmers_summer&winter_internship_영어끝말잇기/Programmers_summer&winter_internship_영어끝말잇기_정세미.java

```diff
@@ -0,0 +1,23 @@
+import java.util.*;
+
+public class Programmers_summer&winter_internship_영어끝말잇기_정세미 {
+    public int[] solution(int n, String[] words) {
+        Set<String> usedWords = new HashSet<>();
+        int wordNum = words.length;
+        char prevAlpha = words[0].charAt(0);
+        int i = 0;
+        for(; i < wordNum; ++i) {
+            String cur = words[i];
+            if(usedWords.contains(cur) || cur.charAt(0) != prevAlpha) {
```

Comment:
```suggestion
        for (; i < wordNum; ++i) {
            String cur = words[i];
            if (usedWords.contains(cur) || cur.charAt(0) != prevAlpha) {
```
컨벤션: 반복문/조건문 소괄호 앞뒤 공백 삽입